documentation/docs/welcome.md

```diff
@@ -1,5 +1,5 @@
 ---
-description: Official IOTA Client Library which can be used to easily integrate an IOTA Client into your application 
+description: Official IOTA Client Library which can be used to easily integrate an IOTA Client into your application
```

Comment:
no eesly :angry:

Reply:
?

Reply:
"Easy" and "simple" are subjective and they are very annoying when it turns out to not be easy or simple in your specific case.